packages/common/src/hooks/useQueryParams/useQueryParams.test.tsx

```diff
@@ -68,7 +68,7 @@ describe('useQueryParams', () => {
     window.resizeTo(1000, 1000);
 
     const { result } = renderHook(
-      () => useQueryParams<TestSortBy>({ key: 'id' }),
+      () => useQueryParams<TestSortBy>({ initialSortBy: { key: 'id' } }),
```

Comment:
just changing the calling signature to be an object `{initialSortBy, initialFilterBy}` rather than just an `initialSortBy`